esp-hal/Cargo.toml

```diff
@@ -91,50 +102,76 @@ jiff = { version = "0.2.10", default-features = false, features = ["static"] }
 [features]
 default = []
 
-bluetooth = []
-
-usb-otg = ["dep:embassy-usb-driver", "dep:embassy-usb-synopsys-otg", "dep:esp-synopsys-usb-otg", "dep:usb-device"]
-
+__bluetooth = []
```

Comment:
Agree with these, but could we get a comment above mentioning that these are private and not covered by semver? Just in case a user doesn't understand this convention.